modules/RTC/RTC.js

```diff
@@ -610,7 +621,10 @@ export default class RTC extends Listenable {
      *
      */
     static getPCConstraints(isP2P) {
-        return isP2P ? RTCUtils.p2pPcConstraints : RTCUtils.pcConstraints;
+        const pcConstraints
+            = isP2P ? RTCUtils.p2pPcConstraints : RTCUtils.pcConstraints;
+
+        return Object.assign({}, pcConstraints);
```

Comment:
why was this needed?  to handle one of the fields not being set at all?

Reply:
To keep the instances kept in RTCUtils from being modified (which can now happen in RTC.createPeerConnection).

Reply:
Note that this will copy nested objects and arrays by reference, not by value. You can use https://lodash.com/docs/#cloneDeep or `JSON.parse(JSON.stringify(obj))` to avoid it.